Tasks/HelmDeployV0/src/helmcommands/helmupgrade.ts

```diff
@@ -46,9 +46,17 @@ export function addArguments(helmCli: helmcli) : void {
         helmCli.addArgument("--force");
     }
 
-    if(valueFile && valueFile != rootFolder) {
-        helmCli.addArgument("--values");
-        helmCli.addArgument("\"" + helmutil.resolvePath(valueFile) + "\"");
+    if (valueFilesInput) {
+        let valueFiles = valueFilesInput.split(/[\n,]+/);
+        valueFiles = tl.findMatch(tl.getVariable('System.DefaultWorkingDirectory') || process.cwd(), valueFiles);
+        if (valueFiles && valueFiles.length > 0) {
+            valueFiles.forEach((file) => {
+                if (file != rootFolder) {
+                    helmCli.addArgument("--values");
+                    helmCli.addArgument("\"" + helmutil.resolvePath(file) + "\"");
+                }
+            });
+        }
```

Comment:
This seems to be duplicate code, add it to utils maybe?

Reply:
Will do